Bahatiroben/PropertyPro-lite
# #167151417 overhauling the UI

#### What does this PR do?
- changes the main colors of the page
- adds a popup up on delete property
- fixes the navigation problem
- adds a forgot password view
-adds pictures to the homepage

#### Description of Task to be completed?
- a user should confirm the delete of property ads
- a user should navigate the pages easily
- a user should reset a password in case of forgot

#### How should this be manually tested?
- clone the repo and switch to branch bg-overhauling-the-ui-167151417
- open the index page
- navigate the pages
- try to delete a property-> you should get a popup asking you for confirmation
- upon Ok -> you get on the  list of all properties
- upon Cancel -> you remainon that page
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#167151417 ] (https://www.pivotaltracker.com/n/projects/2355114)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/39488016/60882570-7b361e00-a248-11e9-906b-17e9275f5ad6.png)

#### Questions:
N/A
